pkgs/applications/virtualization/colima/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, go
```

Comment:
```suggestion
```

Reply:
I assume `go` would be configurable anyway due to `buildGoModule` even though I haven't found an example scanning in `all-packages.nix`, right?

Reply:
Not sure I understand what you're trying to do?

Reply:
Maybe not a use case but I was curious if `go` version of `colima` could be overridden (for the future e.g. if they are behind the nix `go` default version).

I tried `nix-shell -I nixpkgs=$(pwd) -E "{ pkgs ? import <nixpkgs> {} }: let colima = pkgs.colima.override { go = pkgs.go_1_15; }; in pkgs.mkShell { buildInputs = [ colima ]; }"`

Reply:
`buildGo{Module,Package}` is versioned, e.g.

```
  prometheus = callPackage ../servers/monitoring/prometheus {
    buildGoPackage = buildGo115Package;
  };
```

Reply:
I am a blockhead (thanks for you patience)